docs/tutorials/proposals/new-spot-market.md

```diff
@@ -138,18 +141,18 @@ In the tabs below you'll see:
   "rationale": {
```

Comment:
Checked this with [decent.markets](https://decent.markets/tx/#ewoicHJvcG9zYWxTdWJtaXNzaW9uIjogewogICJyYXRpb25hbGUiOiB7CiAgICAiZGVzY3JpcHRpb24iOiAiUHJvcG9zZSBhIHNwb3QgbWFya2V0IGZvciBCaXRjb2luIGFuZCBVU0RUIiwKICAgICJ0aXRsZSI6ICJCaXRjb2luL1VTRFQgc3BvdCBtYXJrZXQiCiAgfSwKICAidGVybXMiOiB7CiAgICAiY2xvc2luZ1RpbWVzdGFtcCI6ICIxNjgzNjI2NTkwIiwKICAgICJlbmFjdG1lbnRUaW1lc3RhbXAiOiAiMTY4MzYyNjYwMCIsCiAgICAibmV3U3BvdE1hcmtldCI6IHsKICAgICAgImNoYW5nZXMiOiB7CiAgICAgICAgImluc3RydW1lbnQiOiB7CiAgICAgICAgICAibmFtZSI6ICJCaXRjb2luIC8gVGV0aGVyIFVTRCAoU3BvdCkiLAogICAgICAgICAgImNvZGUiOiAiQlRDL1VTRFQgc3BvdCIsCiAgICAgICAgICAic3BvdCI6IHsKICAgICAgICAgICAgImJhc2VBc3NldCI6ICJiMzM1Y2Q0YmE4YTljNWIzODdiNjYxMTdlNWVlNmRiZDVhMDNmYjdjNzRlZTRhMWQwMTI1ODlhYWZkNDVlYjI1IiwKICAgICAgICAgICAgInF1b3RlQXNzZXQiOiAiOTQ4OTcwNDgyOTQ2MjQ4ZjBkMDRkZDI3MWQwNjNjZDQ0NDU4ODIyYzY3NjA5ZDIzMGUwNzJkNmQ1MWQ2MDk1NiIKICAgICAgICAgIH0KICAgICAgICB9LAogICAgICAgICJ0aWNrU2l6ZSI6ICIxMCIsICAgICAgICAKICAgICAgICAicHJpY2VEZWNpbWFsUGxhY2VzIjogIjIiLAogICAgICAgICJtZXRhZGF0YSI6IFsKICAgICAgICAgICJiYXNlOkJUQyIsCiAgICAgICAgICAicXVvdGU6VVNEVCIsCiAgICAgICAgICAiY2xhc3M6ZngvY3J5cHRvIiwKICAgICAgICAgICJzcG90IiwKICAgICAgICAgICJzZWN0b3I6ZGVmaSIKICAgICAgICBdLAogICAgCiAgICAgICAgInByaWNlTW9uaXRvcmluZ1BhcmFtZXRlcnMiOiB7CiAgICAgICAgICAidHJpZ2dlcnMiOiBbCiAgICAgICAgICAgIHsKICAgICAgICAgICAgICAiaG9yaXpvbiI6ICIyMTYwMCIsCiAgICAgICAgICAgICAgInByb2JhYmlsaXR5IjogIjAuOTk5OTk5OSIsCiAgICAgICAgICAgICAgImF1Y3Rpb25FeHRlbnNpb24iOiAiODY0MDAiCiAgICAgICAgICAgIH0sCiAgICAgICAgICAgIHsKICAgICAgICAgICAgICAiaG9yaXpvbiI6ICI0MzIwIiwKICAgICAgICAgICAgICAicHJvYmFiaWxpdHkiOiAiMC45OTk5OTk5IiwKICAgICAgICAgICAgICAiYXVjdGlvbkV4dGVuc2lvbiI6ICIzNjAwIgogICAgICAgICAgICB9LAogICAgICAgICAgICB7CiAgICAgICAgICAgICAgImhvcml6b24iOiAiMTQ0MCIsCiAgICAgICAgICAgICAgInByb2JhYmlsaXR5IjogIjAuOTk5OTk5OSIsCiAgICAgICAgICAgICAgImF1Y3Rpb25FeHRlbnNpb24iOiAiMTgwMCIKICAgICAgICAgICAgfSwKICAgICAgICAgICAgewogICAgICAgICAgICAgICJob3Jpem9uIjogIjM2MCIsCiAgICAgICAgICAgICAgInByb2JhYmlsaXR5IjogIjAuOTk5OTk5OSIsCiAgICAgICAgICAgICAgImF1Y3Rpb25FeHRlbnNpb24iOiAiMzAwIgogICAgICAgICAgICB9CiAgICAgICAgICBdCiAgICAgICAgfSwKICAgICAgICAidGFyZ2V0U3Rha2VQYXJhbWV0ZXJzIjogewogICAgICAgICAgInRpbWVXaW5kb3ciOiAiMzYwMCIsCiAgICAgICAgICAic2NhbGluZ0ZhY3RvciI6IDAuMDUKICAgICAgICB9LAogICAgICAgICJzaXplRGVjaW1hbFBsYWNlcyI6ICI0IiwKICAgICAgICAic2xhUGFyYW1zIjogewogICAgICAgICAgInByaWNlUmFuZ2UiOiAiMC4wMyIsCiAgICAgICAgICAiY29tbWl0bWVudE1pblRpbWVGcmFjdGlvbiI6ICIwLjc1IiwKICAgICAgICAgICJwZXJmb3JtYW5jZUh5c3RlcmVzaXNFcG9jaHMiOiAiMSIsCiAgICAgICAgICAic2xhQ29tcGV0aXRpb25GYWN0b3IiOiAiMC44IgogICAgICAgIH0sCiAgICAgICAgImxpcXVpZGl0eUZlZVNldHRpbmdzIjogewogICAgICAgICAgIm1ldGhvZCI6ICJNRVRIT0RfTUFSR0lOQUxfQ09TVCIKICAgICAgICB9LAogICAgICAgICJsb2dOb3JtYWwiOiB7CiAgICAgICAgICAicmlza0F2ZXJzaW9uUGFyYW1ldGVyIjogMWUtMDYsCiAgICAgICAgICAidGF1IjogMy45OTVlLTA2LAogICAgICAgICAgInBhcmFtcyI6IHsKICAgICAgICAgICAgInNpZ21hIjogMS4wCiAgICAgICAgICB9CiAgICAgICAgfQogICAgICB9CiAgICB9CiAgfQogfQp9), it's valid